pkg/ip/ip.go

```diff
@@ -753,15 +752,8 @@ func PartitionCIDR(targetCIDR net.IPNet, excludeCIDR net.IPNet) ([]*net.IPNet, [
 // netip.Addr.Compare (i.e. IPv4 addresses show up before IPv6).
 // The slice is manipulated in-place destructively; it does not create a new slice.
 func KeepUniqueAddrs(addrs []netip.Addr) []netip.Addr {
-	return slices.SortedUniqueFunc(
```

Comment:
Looks like this is removing the only caller of `SortedUniqueFunc`. Should we remove its implementation as well given that benchmarks seem to suggest we should just sort and compact?

Reply:
Yeah fair question - I guess for API symmetry it makes sense to also have `SortedUniqueFunc` given that we have `SortedUnique` (which is used, for better or worse). OTOH, for cases like this one, where the type is `comparable` but not `cmp.Ordered` `SortedUniqueFunc` also doesn't help because it calls `CompactFunc` instead of just `Compact`, even though netip.Addrs are `comparable`.

Given the two different type constraints I'll drop it I think.

(I didn't benchmark `KeepUniqueAddrs` only the sorting funcs.)